code/modules/hydroponics/hydroponics_mutations.dm

```diff
@@ -0,0 +1,529 @@
+//The astrogator has a way with plants of every type.
+//She nurtures alien veg'tables and cooks them when they're ripe!
+//She had a Veged orchid once that ate the captain's cat.
+//Then terrorized the ship's exec, until she squashed it flat!
+
+//Mutates the plant overall (randomly).
+/obj/machinery/portable_atmospherics/hydroponics/proc/mutate(var/severity)
+
+	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\/obj/machinery/portable_atmospherics/hydroponics/proc/mutate() called tick#: [world.time]")
+
+	if(!severity) return
+
+	// No seed? Try to mutate the weeds or pests in the tray, if any.
+	if(!seed)
+		//TODO
+		return
+
+	if(seed.immutable)
+		return
+
+	//Is the plant still a sapling? If so, try to mutate species, otherwise do something bad.
+	if(age < 3)
+		if(seed.mutants. && seed.mutants.len)
+			if(prob(30))
+				mutate_species()
+				return
+		var/mutation_type = pick_mut(severity, "SomethingBad")
+		apply_mut(mutation_type, severity)
+		return
+
+	//writepanic("[__FILE__].[__LINE__] ([src.type])([usr ? usr.ckey : ""])  \\/datum/seed/proc/mutate() called tick#: [world.time]")
```

Comment:
"gee this is everywhere, I don't know what that is but I'd probably better leave everything like it was just in case I get yelled at"